packages/next/build/webpack-config.ts

```diff
@@ -1454,8 +1454,11 @@ export default async function getBaseWebpackConfig(
       ((isServerless && isServer) || webServerRuntime) &&
         new ServerlessPlugin(),
       isServer &&
-        !webServerRuntime &&
```

Comment:
We can’t remove this condition, otherwise the plugin will be executed twice in server compiler and server-web compiler.

Reply:
I added a condition inside pages manifest plugin so it will only emit 1 json for client and 1 js for server web